### PR TITLE
Fix web_resources match patterns

### PR DESCRIPTION
**Pull request by karaggeorge**
Fixes https://github.com/extension-js/extension.js/issues/177

### CODE REVIEW

File: programs/develop/webpack/plugin-extension/feature-web-resources/index.ts [hunk 1]
```diff
@@ -64,7 +65,7 @@ export class WebResourcesPlugin {
               resources: resources.filter(
                 (resource) => !resource.endsWith('.map')
               ),
-              matches
+              matches: cleanMatches(matches),
```
Comment (cezaraugusto) [inline programs/develop/webpack/plugin-extension/feature-web-resources/index.ts:68]: could you write a comment on top of this with the reasoning of why this is needed? this is for future reference.

if you're feeling adventurous, would love to have the first test for the feature-web-resources plugin. It's mainly creating a `__spec__/clean-matches.spec.ts` file with some edge cases. I'm using tests/comments as a reference to populate the new docs.

Comment (karaggeorge) [reply programs/develop/webpack/plugin-extension/feature-web-resources/index.ts:70]: Added a comment and some tests 👍

**Review by cezaraugusto — APPROVED** [review]
Thanks for the patch and tests! Looking great

**Comment by cezaraugusto** [comment]
Available in v2.0.0-alpha.28
